hack/deploy.sh

```diff
@@ -2,4 +2,4 @@
 
 set -e
 
-./hack/build.sh && ytt -f config/ | kbld -f- | kapp deploy -a kc -f- -c -y
+./hack/build.sh && ./hack/ytt-me.sh | kbld -f- | kapp deploy -a kc -f- -c -y
```

Comment:
same goes for deploy.sh as build.sh. we should probably change ./hack/build-release.sh to include a version. @danielhelfand build-release.sh is what gets used for building final artifacts (oss and internal), right?

Reply:
`./hack/build-release.sh` is used for oss but not internal. The internal build is just using a go build, so the ldflag would need to be added along with the version information.

Reply:
huh. -  thanks.  i think that needs a little more thought then.  I appreciated the idea of having the default for dev builds actually keep the develop flag but it sounds like we need to either slightly adjust our internal build process or accept that everything will get versioned.

Reply:
We could adjust the internal build to use the hack script. We basically adopted our internal process from another team and I am not sure why some choices were made. I do appreciate the idea of providing the version build at run time when releasing. It makes it very clear when developing or if someone is building from source that an official release is not being used.